### PR TITLE
feat(TDP-1712): change jest config for more packages

### PR DESCRIPTION
**Pull request by adamosborne-tnl**
Have updated a few more packages to the latest jest-configurator-web

### CODE REVIEW

File: packages/article-comments/__tests__/web/jest.config.js [hunk 1]
```diff
@@ -1,3 +1,13 @@
-const jestConfigurator = require("@times-components/jest-configurator").default;
+const jestConfigurator = require("@times-components/jest-configurator-web")
+  .default;
 
-module.exports = jestConfigurator("web", __dirname);
+  module.exports = jestConfigurator("web", __dirname);
```
Comment (bibicollins1) [inline packages/article-comments/__tests__/web/jest.config.js:4]: Do we still want this bit?

Comment (adamosborne-tnl) [reply packages/article-comments/__tests__/web/jest.config.js:4]: No we don't.... dang, left the web in :-)

File: packages/article-in-depth/__tests__/web/__snapshots__/article.web.test.js.snap [hunk 2]
```diff
@@ -192,19 +192,6 @@ exports[`3. an article with no headline falls back to use shortheadline 1`] = `
         <p>
           Some content
         </p>
-        <div>
```
Comment (bibicollins1) [inline packages/article-in-depth/__tests__/web/__snapshots__/article.web.test.js.snap:195]: Is this meant to be removed?

Comment (adamosborne-tnl) [reply packages/article-in-depth/__tests__/web/__snapshots__/article.web.test.js.snap:195]: yes, had some 'native' elements that was removed

File: packages/article-comments/__tests__/web/jest.config.js (hunk 1 above)
Comment (adamosborne-tnl) [reply packages/article-comments/__tests__/web/jest.config.js:4]: ah, it was a line that should have been deleted. removing now

File: packages/article-in-depth/__tests__/web/__snapshots__/article.web.test.js.snap (hunk 2 above)
Comment (bibicollins1) [reply packages/article-in-depth/__tests__/web/__snapshots__/article.web.test.js.snap:195]: Oh okay. I thought was just the dinomi rail? Or whatever it's called at the bottom. Where did you make the change that removed them?

Comment (adamosborne-tnl) [reply packages/article-in-depth/__tests__/web/__snapshots__/article.web.test.js.snap:195]: so strange, I know i changed a test, this week for that, I will find it

Comment (adamosborne-tnl) [reply packages/article-in-depth/__tests__/web/__snapshots__/article.web.test.js.snap]: I honestly can't find were that was from

Comment (adamosborne-tnl) [reply packages/article-in-depth/__tests__/web/__snapshots__/article.web.test.js.snap]: @bibicollins1 what do you want me to do with this?

**Pull request merged**